### ISSUE TITLE
entity filter doesnt show the entity's

### ISSUE THREAD

**Issue by sebastiaan001** [home-assistant/android#727]
**Home Assistant Android version:**
latest beta

**Android version:**
10
chrome os
**Phone model:**
pocofone f1
onplus nord
chromeos

**Home Assistant version:**
114.4beta
**Last working Home Assistant release (if known):**
x
**Description of problem:**
The entity filter doest show the entity's that are filtert. in the browser they do show up.
**Traceback (if applicable):**

```

```

**Screenshot of problem:**

**Additional information:**
sorry for the poor english

**Comment by dshokouhi** [comment]
Please post a screenshot of the issue you are seeing so we can see whats going on.  Which entity filter are you referring to?

**Comment by sebastiaan001** [comment]
![Screenshot_20200810-231158](https://user-images.githubusercontent.com/43095133/89831957-fd38fa80-db5e-11ea-9e7e-3500364c3b59.jpg)
![Screenshot_20200810-230953](https://user-images.githubusercontent.com/43095133/89831964-00cc8180-db5f-11ea-860e-25973af1ebc3.jpg)

In the first screenshot the States doesn't show. While after a few times reloading they show up (second screenshot) . The States of the lights didn't change. Its a state filter and i filter 'on'

**Comment by dshokouhi** [comment]
Sorry a lil hard to follow.  Are you saying after a few reloads the cards load fine? That almost sounds like a caching issue on your device.  It also looks like you are using a custom card for the lights?

**Comment by sebastiaan001** [comment]
Thanks for your comment, I deleted the custom light cards en that is the fix. the problem was multi device.

**Comment by dshokouhi** [comment]
You probably want to check if the card has an update and if not open an issue with the author :) 

Closing this issue now
